### PR TITLE
psutils update to 3.3.2

### PR DESCRIPTION
**Pull request by HouLiXieBuRou**
Topic Description
-----------------

- psutils: update to 3.3.2

- py: update to 1.11.0
Package pytest relies on the new version of py.psutils->pytest-datafiles->pytest->py

- pytest-datafiles: new,3.0.0
Because the new version of psutils requires

- wand: new,0.6.13
Because the new version of psutils requires

- argparse-manpage: new,4.5
Because the new version of psutils requires

- pypdf: new,4.0.0
Because new psutils need

- puremagic: new,1.15
New version of psutils dependency

Package(s) Affected
-------------------

Security Update?
----------------

No

Build Order
-----------

```
#buildit puremagic pypdf argparse-manpage wand py pytest-datafiles psutils
```

Test Build(s) Done
------------------

**Primary Architectures**

- [ ] AMD64 `amd64`
- [ ] AArch64 `arm64`

**Secondary Architectures**

- [ ] Loongson 3 `loongson3`
- [ ] PowerPC 64-bit (Little Endian) `ppc64el`
- [ ] RISC-V 64-bit `riscv64`

**Experimental Architectures**

- [ ] MIPS R6 64-bit (Little Endian) `mips64r6el`

### CODE REVIEW

**Comment by HouLiXieBuRou** [comment]
psutils bin
image

File: lang-python/pypdf/spec [hunk 1]
```diff
@@ -0,0 +1,4 @@
+VER=4.0.0
+SRCS="tbl::https://github.com/py-pdf/pypdf/archive/refs/tags/$VER.tar.gz"
```
Comment (MingcongBai) [inline lang-python/pypdf/spec]: Use Git tag.

File: lang-python/py/spec [hunk 2]
```diff
@@ -1,6 +1,4 @@
-VER=1.8.1
-SRCS="tbl::https://github.com/pytest-dev/py/archive/$VER.tar.gz"
-CHKSUMS="sha256::84fcea6e4851cb614e6f71bb462c8c6046c2e81065accaedb6d1797d104c8994"
-SUBDIR="py-$VER"
-REL=2
+VER=1.11.0
+SRCS="tbl::https://github.com/pytest-dev/py/archive/refs/tags/$VER.tar.gz"
```
Comment (MingcongBai) [inline lang-python/py/spec]: Use Git tag.

File: lang-python/argparse-manpage/spec [hunk 3]
```diff
@@ -0,0 +1,4 @@
+VER=4.5
+SRCS="tbl::https://github.com/praiskup/argparse-manpage/archive/refs/tags/v$VER.tar.gz"
```
Comment (MingcongBai) [inline lang-python/argparse-manpage/spec]: Use Git tag.

File: lang-python/wand/spec [hunk 4]
```diff
@@ -0,0 +1,4 @@
+VER=0.6.13
+SRCS="tbl::https://github.com/emcconville/wand/archive/refs/tags/$VER.tar.gz"
```
Comment (MingcongBai) [inline lang-python/wand/spec]: Use Git tag.

File: lang-python/pytest-datafiles/spec [hunk 5]
```diff
@@ -0,0 +1,4 @@
+VER=3.0.0
+SRCS="https://github.com/omarkohl/pytest-datafiles/archive/refs/tags/3.0.0.tar.gz"
```
Comment (MingcongBai) [inline lang-python/pytest-datafiles/spec]: Use Git tag.

**Review by MingcongBai — CHANGES_REQUESTED** [review]

File: app-doc/psutils/autobuild/prepare [hunk 6]
```diff
@@ -0,0 +1,6 @@
+# FIXME:The appearance of other directories in the root directory of this source code package will affect the automatic detection of setuptools, leading to build failure, and the upstream is unwilling to modify it. And if the source code package is processed according to the PEP517 standard, it will make an error and cannot find itself, so do this processing first
+mkdir "${SRCDIR}"/.tempsrc
```
Comment (MingcongBai) [inline app-doc/psutils/autobuild/prepare]: Please use absolute paths.

**Review by MingcongBai — CHANGES_REQUESTED** [review]

**Review by MingcongBai — APPROVED** [review]

**Pull request merged**